openstack_hypervisor/hooks.py

```diff
@@ -1643,6 +1648,46 @@ def configure(snap: Snap) -> None:
     _setup_secrets(snap)
     _detect_compute_flavors(snap)
 
+    cpu_shared_set, allocated_cores = _get_cpu_pinning_context(snap)
```

Comment:
Can this be moved to _get_configure_context instead of passing values to the function